src/sentry/static/sentry/app/components/events/meta/metaProxy.tsx

```diff
@@ -52,23 +56,25 @@ export class MetaProxy {
   }
 }
 
-export function withMeta(event) {
+export function withMeta<T>(event: T): T {
```

Comment:
the return type here would be of type `T | T[]` but you can remove it, it will be automatically set by typescript

Reply:
we never return multiple Ts, what do you mean?